xclim/ensembles/_reduce.py

```diff
@@ -113,6 +113,9 @@ def _make_crit(da):
     else:
         # Easy peasy, skip all the convoluted stuff
         crit = _make_crit(ds)
+
+    # drop criteria that are all NaN
+    crit = crit.dropna(dim="criteria", how="all")
```

Comment:
Can you mention this change in the docstring ? For example, in the notes we show how to retrieve the original array, maybe you could add a mention that this will not be possible if there are all-nan criteria ?